todo_app/app.py

```diff
@@ -1,11 +1,32 @@
 from flask import Flask
+from flask import render_template
+from flask import request
+from flask import redirect
```

Comment:
Not an issue, but worth noting if you weren't aware that we can combine these all;
```suggestion
from flask import render_template, request, redirect
```